3.0/vector_tile.proto

```diff
@@ -45,38 +45,41 @@ message Tile {
                 // section 4.3 of the specification.
                 repeated uint32 geometry = 4 [ packed = true ];
                 
-                // Properties replace existing tags field and
-                // uses the properties field instead. This would only be used if version
-                // for a layer is 3 or greater and tags should not be used at that point
+                // Additional tags (or all the tags) of this feature may be
+                // encoded as repeated pairs of 32-bit integers, to take
```

Comment:
Didn't we want to get away from the "tags" name and use "properties" instead? Also the `properties` field has 64bit uints, not 32 bit ints. And this is not necessarily "pairs" when we deal with lists and maps.